packages/time-picker/src/TimePicker/components/Popover/Popover.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+import {
+  useOverlay,
+  AriaOverlayProps,
+  DismissButton,
+} from "@react-aria/overlays"
+import { FocusScope } from "@react-aria/focus"
+import styles from "./Popover.module.scss"
+
+export const Popover: React.FunctionComponent<
```

Comment:
```suggestion
export const Popover: React.VFC<
```

As with all the other components (I know it's deprecated in React 18, but since we still support 16 and 17, we haven't changed it yet).